src/App.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList/TodoList';
+import todos from './api/todos';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import users from './api/users';
+import { PreparedTodo } from './types/PreparedTodo';
+
+const preparedTodosArr: PreparedTodo[] = todos.map(todo => {
+  const user = users.find(person => person.id === todo.userId) || null;
+
+  return { ...todo, user };
+});
 
 const App: React.FC = () => (
   <div className="App">
-    <h1>Static list of todos</h1>
+    <TodoList preparedTodos={preparedTodosArr} />
```

Comment:
just todos, avoid prepared word here
```suggestion
    <TodoList todos={preparedTodos} />
```